Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithmFunction.cpp

```diff
@@ -19,11 +19,18 @@
 
 namespace {
 
-template <typename track_finder_t>
+using Updater = Acts::GainMatrixUpdater;
+using Smoother = Acts::GainMatrixSmoother;
+
+using Stepper = Acts::EigenStepper<>;
+using Navigator = Acts::Navigator;
+using Propagator = Acts::Propagator<Stepper, Navigator>;
```

Comment:
This is a bit of a problem with this PR, these typedefs are largely not needed anymore, since the `EigenStepper` type becomes much shorter in almost all cases. I wasn't sure if I should try to remove these in this PR or leave that to be done in a future PR.

Reply:
I would suggest we do a follow-up PR, where we clean this out then. @XiaocongAi would you have time to do this?